### PR TITLE
Name goreleaser builds based off tag, not version so that the v is appended

### PR DESCRIPTION
**Pull request by nikkictl**
Signed-off-by: Nikki Attea <nikki@sensu.io>

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -24,9 +25,7 @@ builds:
       - 7
 
 checksum:
-  # You can change the name of the checksums file.
-  # Default is `{{ .ProjectName }}_{{ .Version }}_checksums.txt`.
-  name_template: "{{ .ProjectName }}_{{ .Version }}_sha256-checksums.txt"
+  name_template: "{{ .ProjectName }}_{{ .Tag }}_sha256-checksums.txt"
```
Comment (palourde) [inline .goreleaser.yml:28]: Could you just replace `sha256` with `sha512`? It's my mistake, I probably copied/pasted from 2 different sources 😬

**Review by palourde — APPROVED** [review]
👌

File: .goreleaser.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ builds:
     main: main.go
     # Set the binary output location to bin/ so archive will comply with Sensu Go Asset structure
     binary: bin/{{ .ProjectName }}
+    name_template: "{{ .ProjectName }}-{{ .Tag }}"
```
Comment (palourde) [inline .goreleaser.yml:9]: This line appears to be invalid at this location, goreleaser threw me this error:

```
line 9: field name_template not found in type config.Build
```

**Review by palourde — CHANGES_REQUESTED** [review]

File: .goreleaser.yml (hunk 2 above)
Comment (palourde) [reply .goreleaser.yml:9]: Update: I believe we actually need to set `name_template` in the `archives` section:

```
archives:
  - id: tar
    format: tar.gz
    name_template: "{{ .ProjectName }}_{{ .Tag }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
[...]
```

**Comment by palourde** [comment]
Update: As discussed with Sean, let's scrap the `v` prefix for plugins versions.

**Comment by nikkictl** [comment]
Closing without merge!

**Pull request closed without merging**